lib/views/go_to_page.dart

```diff
@@ -148,13 +164,76 @@ class _GoToPageState extends State<GoToPage> {
                       },
                       onSuggestionSelected: (suggestion) {
                         this._destinTypeAheadController.text = suggestion;
-                        originDestination.add("$suggestion");
-                        widget.route(originDestination);
+                        FocusScope.of(context).requestFocus(
+                            destinationTextField); // to change the focus of the textfield to the second textfield
+                        for (RoomCoordinate room in roomsList) {
+                          if (room.roomId == suggestion) {
+                            originDestinationCoords[1] = room;
+                          }
+                        }
                       },
                     ),
                   ),
                 ],
               ),
+              SizedBox(height: 20.0),
+              Center(
+                child: FloatingActionButton(
+                  child: Icon(
+                    Icons.navigation,
+                    color: Colors.white,
+                    size: 40.0,
+                  ),
+                  backgroundColor: Color.fromRGBO(147, 0, 47, 1),
+                  elevation: 2,
+                  onPressed: () {
+                    print("THIS LISTTT IS longgg" + originDestinationCoords.length.toString());
```

Comment:
What's the purpose of this print statement? Can this be removed?

Reply:
Fixed!